health-services/individual/src/main/java/org/egov/individual/service/IndividualService.java

```diff
@@ -415,4 +413,20 @@ private void integrateWithUserService(IndividualBulkRequest request,
             }
         }
     }
+    Boolean isSmsEnabledForRole(IndividualRequest request) {
+        List<String> roleCodes = new ArrayList<>();
+        if (StringUtils.isEmpty(properties.getSmsDisabledRoles()))
```

Comment:
These should be collection utils, not stringutils also keep the if condition as the first line